internal/model/deputy.go

```diff
@@ -21,3 +28,11 @@ func (d Deputy) GetURL() string {
 func (d Deputy) IsPro() bool {
 	return d.Type.Handle == "PRO"
 }
+
+func (d Deputy) CalculateNonCompliance() string {
```

Comment:
From the other PR, aren't we doing this in the query itself?

```CASE WHEN COALESCE(di.activeClientCount,0) = 0 THEN 0
                    ELSE CAST(di.activeNonCompliantClientCount AS decimal(7,2)) / CAST(di.activeClientCount AS decimal(7,2))
                    END AS nonComplianceScore```

Reply:
Yeah, the query calculates it as a decimal for sorting purposes, but this here makes it pretty for displaying on the front-end

Reply:
(shown as a percentage rounded to the nearest integer)